yandextank/plugins/Autostop/criterions.py

```diff
@@ -20,7 +20,7 @@ def get_type_string():
     def __init__(self, autostop, param_str):
         AbstractCriterion.__init__(self)
         self.seconds_count = 0
-        params = param_str.split(',')
+        params = param_str.split(',', 3)
```

Comment:
Should it be 2, not 3?
In case of 2 as a parameter ```self.tag``` would include the rest of the string with commas,
in case of 3 all these trailing commas will be dropped.
```param_str = 'ms,s,long tag, with commas'

params_3 = param_str.split(',', 3)
tag = params_3[2].strip() if len(params_3) == 3 else None
print(tag)
None

params_2 = param_str.split(',', 2)
tag = params_2[2].strip() if len(params_2) == 3 else None
print(tag)
'long tag, with commas'

Reply:
@Tonsofattraction could you comment this, please?

Reply:
My mistake, this particular criterion accepts 3 params at most. May be we should check other criterions as well.